src/amp/components/AnalyticsIframe.tsx

```diff
@@ -1,9 +1,6 @@
 import React from 'react';
 import { css } from 'emotion';
 
-const prebidSrc =
```

Comment:
By removing this, does it remove prebid completely? Or has this been added in again elsewhere?

Reply:
We're pulling this from the config as a prop like `<AdUserSync url={config.ampIframeUrl} />`